### PR TITLE
update to n-ui v3

### PR DESCRIPTION
**Pull request by gvonkoss**
@wheresrhys @adgad does this look about right?

### CODE REVIEW

**Comment by wheresrhys** [comment]
Hold fire for a bit - I uncovered a bug

**Comment by wheresrhys** [comment]
by uncovered, I mean introduced :joy:

**Comment by wheresrhys** [comment]
scratch that... uncovered. I'm too hard on myself sometimes

**Comment by keirog** [comment]
Shouldn’t this bump n-ui in bower too?

**Comment by gvonkoss** [comment]
Yep, I'm working on all of dem conflicts

File: client/main.scss [hunk 1]
```diff
@@ -1,9 +1,6 @@
-$output-critical: true !default;
-
-
+$n-ui-output-critical: true;
```
Comment (wheresrhys) [inline client/main.scss]: probably need to add myftHint and myftDigestPromo as critical styles (speak to @pixelandpage )

**Pull request merged**